spec/features/responsible_body/devices_setup_spec.rb

```diff
@@ -408,7 +408,7 @@ def when_i_select_orders_will_be_placed_centrally
   end
 
   def given_there_is_a_school_with_no_standard_device_allocation
-    expect(@zebra_school.has_laptop_allocation?).to be false
+    expect(@zebra_school.laptops_allocated?).to be false
```

Comment:
If this had stayed as `has_laptop_allocation?` you would have the RSpec `have_laptop_allocation` matcher

Reply:
You are right but the matcher was not used in the spec at all. 
I renamed it to be shorter and easier to type. I don´t mind to revert it back to its original name but would it work for you if I rename it to `laptop_allocated?` and in the spec do this?:

`expect(@zebra_school).not_to be_laptop_allocated`

Reply:
It's alright, I'd leave it